### PR TITLE
Feature/3642 remove js templates

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # BLUI-3642

#### Changes proposed in this Pull Request:
- Removed javascript templates 

#### To Test:
- Please refer to testing locally instructions in Readme
- npx create-react-app myapp --template file:../path/to/template
- Templates available now: 
blank-typescript
routing-typescript
authentication-typescript

**Note: DO NOT MERGE until https://github.com/brightlayer-ui/brightlayer-ui-cli/pull/144 gets merge**

_**In future, we can rename the templates from blank-typescript to blank, but for now lets use the same**_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,56 +5,65 @@ These are project templates that are used by Create React App when starting a ne
 ## Available Templates
 
 ### Blank
-The Blank template provides a bare-bones application with Brightlayer UI components and themes automatically configured and ready to use. This template is available in [TypeScript](https://www.npmjs.com/package/@brightlayer-ui/cra-template-blank-typescript)(recommended) and [JavaScript](https://www.npmjs.com/package/@brightlayer-ui/cra-template-blank).
+
+The Blank template provides a bare-bones application with Brightlayer UI components and themes automatically configured and ready to use. This template is available in [TypeScript](https://www.npmjs.com/package/@brightlayer-ui/cra-template-blank-typescript).
```
Comment (daileytj) [inline README.md:9]:
```suggestion
The [Blank template](https://www.npmjs.com/package/@brightlayer-ui/cra-template-blank-typescript) provides a bare-bones application with Brightlayer UI components and themes automatically configured and ready to use.
```

File: README.md [hunk 2]
```diff
@@ -5,56 +5,65 @@ These are project templates that are used by Create React App when starting a ne
...
 
 ### Routing
-The Routing template provides all of the essential Brightlayer UI configuration as well as additional configuration for using using [React Router](https://reactrouter.com/). It also adds a navigation Drawer to switch between several sample screens. This template is available in [TypeScript](https://www.npmjs.com/package/@brightlayer-ui/cra-template-routing-typescript)(recommended) and [JavaScript](https://www.npmjs.com/package/@brightlayer-ui/cra-template-routing).
+
+The Routing template provides all of the essential Brightlayer UI configuration as well as additional configuration for using using [React Router](https://reactrouter.com/). It also adds a navigation Drawer to switch between several sample screens. This template is available in [TypeScript](https://www.npmjs.com/package/@brightlayer-ui/cra-template-routing-typescript).
```
Comment (daileytj) [inline README.md:13]:
```suggestion
The [Routing template](https://www.npmjs.com/package/@brightlayer-ui/cra-template-routing-typescript) provides all of the essential Brightlayer UI configuration as well as additional configuration for using using [React Router](https://reactrouter.com/). It also adds a navigation Drawer to switch between several sample screens.
```

File: README.md [hunk 3]
```diff
@@ -5,56 +5,65 @@ These are project templates that are used by Create React App when starting a ne
...
 
-|                | placeholder landing page | navigation drawer & routing  | login & registration screens  |
-| -------------- | -- | -- | -- |
-| Blank          | ✅ |    |    |
-| Routing        | ✅ | ✅ |    |
-| Authentication | ✅ | ✅ | ✅ |
+The Authentication template includes everything from the Routing template and also installs and configures the Brightlayer UI [react-auth-workflow](https://www.npmjs.com/package/@brightlayer-ui/react-auth-workflow) package to automatically wrap the main application with functions and screens for Login, Registration, Change Password, etc. This template is available in [TypeScript](https://www.npmjs.com/package/@brightlayer-ui/cra-template-authentication-typescript).
```
Comment (daileytj) [inline README.md:17]:
```suggestion
The [Authentication template](https://www.npmjs.com/package/@brightlayer-ui/cra-template-authentication-typescript) includes everything from the Routing template and also installs and configures the Brightlayer UI [react-auth-workflow](https://www.npmjs.com/package/@brightlayer-ui/react-auth-workflow) package to automatically wrap the main application with functions and screens for Login, Registration, Change Password, etc.
```

**Review by daileytj — COMMENTED** [review]
Just a few changes

File: README.md [hunk 4]
```diff
@@ -5,56 +5,65 @@ These are project templates that are used by Create React App when starting a ne
...
 ##### Login Screen from Auth Template
+
 ![Authentication Template Login](./images/authentication.png)
 
 ## Usage
+
 You can use these templates with the Brightlayer UI CLI (recommended):
 
 ```sh
-npx -p @brightlayer-ui/cli blui new react --template=blank
```
Comment (joebochill) [inline README.md:35]: Are we still able to accept this shorthand template name, but use the typescript implementation of the template (i.e., template=blank still works, but uses the typescript template)?

Comment (bkarambe) [reply README.md:35]: Yes we are still using javascript template's shorthand and create typescript template. If I provide --template=blank it creates blank-typescript project

**Comment by bkarambe** [comment]
@joebochill @daileytj @EvanMcLaughlin-eaton @jeffvg @huayunh Please review.

**Review by daileytj — APPROVED** [review]

**Pull request merged**